docs/content/commands/rclone.md

```diff
@@ -649,39 +649,6 @@ rclone [flags]
       --quatrix-maximal-summary-chunk-size SizeSuffix       The maximal summary for all chunks. It should not be less than 'transfers'*'minimal_chunk_size' (default 95.367Mi)
       --quatrix-minimal-chunk-size SizeSuffix               The minimal size for one chunk (default 9.537Mi)
       --quatrix-skip-project-folders                        Skip project folders in operations
-  -q, --quiet                                               Print as little stuff as possible
```

Comment:
Not sure where this change came from?

This is in an auto generated file so you should probably drop this change and the other changes in the auto generated files. (They all have the string `autogenerated - DO NOT EDIT` in them)